jenkins/kernel-defconfig-creator.sh

```diff
@@ -116,15 +100,10 @@ fi
 
 # Tree specific fragments: stable
 if [ ${TREE_NAME} = "stable" ] || [ ${TREE_NAME} = "stable-rc" ]; then
-  # Don't do allmodconfig builds
-  DEFCONFIG_LIST=${DEFCONFIG_LIST/allmodconfig/}
```

Comment:
From a quick grok of the rest of this file, removing this line here would mean that allmodconfig is now built for x86 and arm64 for stable and stable-rc.
If we actually want to do that, you might as well remove this IF section entirely as it does nothing

Reply:
The line removed here was already pointless because the following line:

  # only do minimal "known stable" defconfigs  
  DEFCONFIG_LIST=$DEFCONFIG_STABLE 

rendered it moot, because DEFCONFIG_STABLE doesn't have allmodconfig in it anyways.